lib/auth/sessions.go

```diff
@@ -363,6 +363,17 @@ func (a *Server) newWebSession(
 	if err != nil {
 		return nil, nil, trace.Wrap(err)
 	}
+
+	if tdr, err := a.calculateTrustedDeviceMode(ctx, func() ([]types.Role, error) {
+		return checker.Roles(), nil
+	}); err != nil {
+		log.
+			WithError(err).
+			Warnf("Failed to calculate trusted device mode for session")
```

Comment:
nit: Nothing to format.

```suggestion
			Warn("Failed to calculate trusted device mode for session")
```